digital_land/log.py

```diff
@@ -156,3 +156,43 @@ def add(self):
     def save(self, *args, **kwargs):
         self.add()
         super().save(*args, **kwargs)
+
+
+class ConvertedResourceLog(Log):
+    fieldnames = [
+        "dataset",
+        "resource",
+        "command",
+        "env",
+        "gdal-version",
+        "return-code",
+        "stdout",
+        "stderr",
```

Comment:
Worth checking through the code to see if gdal is always used I think there are some where it's not.

Is it worth attempting to copy how downloading works a bit in terms of catching exceptions rather than stdout and stderr, they. can be included int he exception (we can create a special exception).

could be worth including the input format and maybe a result column so we know if it has. or hasn't been converted